compiler/rustc_middle/src/ty/fast_reject.rs

```diff
@@ -60,29 +60,23 @@ pub enum StripReferences {
     No,
 }
 
-/// Tries to simplify a type by only returning the outermost injective¹ layer, if one exists.
+/// Tries to simplify a type by only returning the outermost layer of an inductive type, if one exists.
```

Comment:
I don't really know what "inductive" type means here, and I expect other readers won't either =)